query/command_graphite_test.go

```diff
@@ -0,0 +1,373 @@
+// Copyright 2015 Square Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Integration test for the query execution.
+package query
+
+import (
+	"testing"
+
+	"github.com/square/metrics/api"
+	"github.com/square/metrics/api/backend"
+	"github.com/square/metrics/assert"
+)
+
+type fakeGraphiteAPI struct {
```

Comment:
i think you can do the same thing here - embed the interface, and don't implement some of the methods that are never called in the tests.